docs/source/app-dev/grpc/daml-to-ledger-api.rst

```diff
@@ -34,13 +34,15 @@ Records or product types are translated to :ref:`com.daml.ledger.api.v1.record`.
 
 .. literalinclude:: ../code-snippets/Types.daml
 	:language: daml
-	:lines: 9-18
+	:start-after: -- PRODUCT_TYPE_DEF_BEGIN
+	:end-before: -- PRODUCT_TYPE_DEF_END
 
 And here's an example of creating a value of type `MyProductType`:
 
 .. literalinclude:: ../code-snippets/Types.daml
 	:language: daml
-	:lines: 29,31,33-41
```

Comment:
I'm not sure why some lines were skipped here. My version simply includes all the lines before `let myProduct` so it's clear where the variables come from.